tests/runtests.py

```diff
@@ -409,6 +415,14 @@ def paired_tests(paired_test, options, test_labels):
         '--parallel', dest='parallel', nargs='?', default=0, type=int,
         const=default_test_processes(),
         help='Run tests in parallel processes.')
+    parser.add_argument(
+        '--tag', dest='tags', action='append',
+        help='Tells Django to only run tests having specified tags. You can '
```

Comment:
See comments above on public test command arg descriptions; these should probably just be kept in sync with those.